pcdsdevices/lxe.py

```diff
@@ -241,3 +245,18 @@ def __init__(self, prefix='', *, egu=None, **kwargs):
                 f' of seconds.'
             )
         super().__init__(prefix, egu='s', **kwargs)
+
+    def set_current_position(self, position):
+        '''
+        Calculate and configure the user_offset value, indicating the provided
+        ``position`` as the new current position.
+
+        Parameters
+        ----------
+        position
+            The new current position.
+        '''
+        self.user_offset.put(0.0)
+
+        new_offset = self.setpoint.get() - position
```

Comment:
Tbh, I was a little confused by your 'added when setting, subracted when reading' note but it makes sense to me now that I'm looking at it with a clear mind and this equation helps to clarify.

Is this convention consistent with EpicsMotor offsets and motor record offsets? Personally, I would expect the offset to work the other way around: Dial + Offset = RBV

Reply:
I think you're probably right. When there's two options like this, I generally get it wrong 😆 

Checking the motor record docs: `userVAL = DialVAL * DIR + OFFset`
Indicates that: `user value = dial value + offset`
And if my advanced algebra skills haven't failed me: `dial value = user_value - offset`

I'll mark this as draft again and fix it up. Thanks for being thorough here 👍